lms/envs/common.py

```diff
@@ -1529,8 +1529,16 @@
 
 
 common_js = set(rooted_glob(COMMON_ROOT / 'static', 'coffee/src/**/*.js')) - set(courseware_js + discussion_js + notes_js + instructor_dash_js)    # pylint: disable=line-too-long
-project_js = set(rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/**/*.js')) - set(courseware_js + discussion_js + notes_js + instructor_dash_js)  # pylint: disable=line-too-long
-
+project_js = (
+    set(
+        rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/**/*.js') +
+        [
+            'common/js/xblock/runtime.v1.js',
+            'lms/js/xblock/lms.runtime.v1.js',
+        ]
+    )
+    - set(courseware_js + discussion_js + notes_js + instructor_dash_js)
```

Comment:
Yeah, this logic is gross and only getting worse. I don't like introducing variables that are only used in one place, but I hear you on the complexity of this statement. It doesn't help that ``project_js`` is also a meaningless name. :-)